multires-lyrics-search/README.md

```diff
@@ -1,109 +1,176 @@
 
-# Lyrics Live Search: Understanding the Concept of Chunk
-
-[![Jina](https://github.com/jina-ai/jina/blob/master/.github/badges/jina-badge.svg?raw=true  "We fully commit to open-source")](https://get.jina.ai)
+# Lyrics Live Search: Understanding the Concept of Chunks
```

Comment:
Can we simplify? Like `Lyrics Live Search: Understanding Chunks`

Reply:
Done